src/Kernel32.Shared/Kernel32.Helpers.cs

```diff
@@ -47,5 +48,60 @@ public static Win32ErrorCode GetLastError()
         {
             return (Win32ErrorCode)Marshal.GetLastWin32Error();
         }
+
+        public static unsafe uint WriteFile(SafeObjectHandle hFile, void* lpBuffer, uint nNumberOfBytesToWrite)
```

Comment:
Sounds good. Adding a code comment explaining this may avoid folks "enhancing" your code then so that it stays as you intended.